package.json

```diff
@@ -125,6 +125,7 @@
     "karma-time-stats-reporter": "0.1.0",
     "lint-staged": "16.1.5",
     "lodash.isequal": "4.5.0",
+    "ng-mocks": "14.13.5",
```

Comment:
not sure if it's a good idea to add this and, if so, let's do it in another PR anyway